docs/UserGuide.md

```diff
@@ -130,23 +130,19 @@ Examples:
 
 ### Locating persons by attribute: `find`
 
-Finds persons whose attributes contain any of the given keywords.
+Need to quickly find a client by their address or something other than their name? Use this command to find clients whose attributes contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [t/TAG]…​`
 
-* The search is case-insensitive. e.g. `hans` will match `Hans`
-* Keywords are matched against Name, Phone, Address, Email, Birthday, Tag
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
-* Partial matches of a keyword will also be returned. e.g. ``9123`` will return person
-with the phone number ``91234567``
+* The search is case-insensitive. e.g. `hans` will match `Hans`.
+* At least one of the optional fields must be provided.
+* Keywords are matched against the attribute that is inputted (i.e. searching for names will match against all names).
+* Partial matches of a keyword will also be returned. e.g. ``9123`` will return clients
+with the phone number ``91234567``.
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-* `find serangoon` returns ``Bernice Yu``, ``David Li``
+* `find n/John` returns clients with names `john` and `John Doe`.
+* `find a/serangoon` returns ``Bernice Yu``, ``David Li``.
 ![result for 'find serangoon'](images/findSerangoonResult.png)
 
 ### Deleting a person : `delete`
```

Comment:
Perhaps we could change this to client as well?

Reply:
Currently only changing the commands I uploaded. Let's update again in the next version